auth/handlers/handle_sessions.go

```diff
@@ -99,14 +106,42 @@ func (h *AuthHandlers) sessionsProc(req *request.AuthReq) error {
 	return nil
 }
 
+func (h *AuthHandlers) sessionsDeleteByUser(req *request.AuthReq) error {
+	userID, err := strconv.ParseUint(req.Request.PostFormValue("userID"), 10, 64)
+	if userID == 0 {
+		return err
+	}
+
+	ss, err := h.getSessionsByUser(req.Context(), userID, userSessionParams{})
+	if err != nil {
+		return err
+	}
+
+	for _, s := range ss {
+		if err = h.SessionManager.DeleteByID(req.Context(), s.id); err != nil {
```

Comment:
how about if we just use DeleteByUserID() instead of loading+looping+deleting-each-one?